fuzzylite/CMakeLists.txt

```diff
@@ -230,10 +265,19 @@ if(FL_BUILD_TESTS)
 endif()
 
 ###INSTALL SECTION
+
 if(NOT FL_INSTALL_BINDIR)
     set(FL_INSTALL_BINDIR bin)
 endif()
 
+if(NOT FL_INSTALL_INCLUDEDIR)
+    if(NOT CMAKE_INSTALL_INCLUDEDIR)
```

Comment:
Could you swap the negation to use positive first?
``` if(CMAKE_INSTALL_INCLUDEDIR)
 +        set(FL_INSTALL_INCLUDEDIR ${CMAKE_INSTALL_INCLUDEDIR})
 +    else()
 +        set(FL_INSTALL_INCLUDEDIR include)
 +    endif()
 +endif()
```

Also, should the `include` not also append the `fl` path to the headers? I am concerned that the headers will be dropped in the `include` folder instead of in a self-contained `fl` folder within `include`.

Reply:
Done. And to answer your question: not at this stage. This is only to define the install prefix. Headers are installed to a subdirectory later on:

    install(DIRECTORY fl/ DESTINATION ${FL_INSTALL_INCLUDEDIR}/fl)